docs/README-Extending.md

```diff
@@ -176,10 +176,60 @@ if (zoweExplorerApi) {
   zoweExplorerApi.registerUssApi(new FtpUssApi());
   zoweExplorerApi.registerMvsApi(new FtpMvsApi());
   zoweExplorerApi.registerJesApi(new FtpJesApi());
-
+}
```

Comment:
@traeok shouldn't this closing bracket be after await zoweExplorerApi.getExplorerExtenderApi().reloadProfiles();